src/personio_py/models.py

```diff
@@ -588,19 +588,32 @@ def __init__(self,
         self.start_date = start_date
         self.end_date = end_date
         self.days_count = days_count
-        self.half_day_start = half_day_start
-        self.half_day_end = half_day_end
+        self.half_day_start = bool(half_day_start)
+        self.half_day_end = bool(half_day_end)
         self.time_off_type = time_off_type
         self.employee = employee
         self.created_by = created_by
         self.certificate = certificate
         self.created_at = created_at
 
-    def _create(self, client: 'Personio'):
-        pass
-
-    def _delete(self, client: 'Personio'):
-        pass
+    def _create(self, client: 'Personio' = None):
+        return get_client(self, client).create_absence(self)
```

Comment:
I think it would be cleaner to have `get_client()` as a method of `PersonioResource` instead of a static function

Reply:
Changing the get_client function also alters code not altered by this PR. Maybe the change would better be handled in a separate pr?

Reply:
sure, we can split this off